components/p3a/nitro_utils/attestation.cc

```diff
@@ -35,16 +36,28 @@ namespace nitro_utils {
 
 namespace {
 
-constexpr char kHashPrefix[] = "sha256:";
-constexpr size_t kHashPrefixLength = 7;
-constexpr size_t kSHA256HashLength = 32;
-constexpr size_t kUserDataMinLength = kSHA256HashLength + kHashPrefixLength;
 constexpr size_t kAttestationBodyMaxSize = 16384;
+// The user_data field contains a multihash of the TLS cert
+constexpr size_t kMultihashPrefixLength = 2;
+constexpr size_t kMultihashSHA256Code = 0x12;
```

Comment:
It might be nice to add https://github.com/multiformats/multicodec/blob/b98f2f38fc63c992545263815c2afdd692d2dc8e/table.csv#L9 as a comment here

Reply:
Addressed in 9b1cfccaca47fded6486b47f3fb306764d9e284c